app/src/main/java/com/example/issueTalk/feature/auth/LoginViewModel.kt

```
@@ -0,0 +1,89 @@
+            try {
+                val result =
+                    auth.signInWithEmailAndPassword(
+                        emailText.value.trim(),
+                        passwordText.value.trim()
+                    ).await()
+
+
+                if (result.user != null) {
+                    _eventChannel.send(LoginEvent.NavigateToHome)
+                }
+            } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)